frontend/src/employee-frontend/components/person-profile/PersonFridgeChild.tsx

```diff
@@ -109,60 +111,70 @@ export default React.memo(function PersonFridgeChild({ id, open }: Props) {
-                        toggleUiMode(`remove-fridge-child-${fridgeChild.id}`)
-                      }}
-                      conflict={fridgeChild.conflict}
-                      deletableFor={
-                        fridgeChild.conflict
-                          ? ['ADMIN', 'FINANCE_ADMIN', 'UNIT_SUPERVISOR']
-                          : ['ADMIN', 'FINANCE_ADMIN']
-                      }
-                    />
-                  </ButtonsTd>
-                </Tr>
-              ))}
+              ).map(
+                (
+                  {
+                    data: fridgeChild,
+                    permittedActions
+                  }: ParentshipWithPermittedActions,
+                  i: number
+                ) => (
+                  <Tr
+                    key={`${fridgeChild.child.id}-${i}`}
+                    data-qa="table-fridge-child-row"
+                  >
+                    <NameTd>
+                      <Link to={`/child-information/${fridgeChild.child.id}`}>
+                        {formatName(
+                          fridgeChild.child.firstName,
+                          fridgeChild.child.lastName,
+                          i18n,
+                          true
+                        )}
+                      </Link>
+                    </NameTd>
+                    <Td>
+                      {fridgeChild.child.socialSecurityNumber ??
+                        fridgeChild.child.dateOfBirth.format()}
+                    </Td>
+                    <Td data-qa="child-age">
+                      {getAge(fridgeChild.child.dateOfBirth)}
+                    </Td>
+                    <DateTd>{fridgeChild.startDate.format()}</DateTd>
+                    <DateTd>{fridgeChild.endDate.format()}</DateTd>
+                    <ButtonsTd>
+                      <Toolbar
+                        dateRange={fridgeChild}
+                        onRetry={
+                          fridgeChild.conflict
+                            ? () => {
+                                void retryParentship(fridgeChild.id).then(
+                                  reloadFridgeChildren
+                                )
+                              }
+                            : undefined
+                        }
+                        editable={permittedActions.includes('UPDATE')}
+                        onEdit={() => {
+                          setSelectedParentshipId(fridgeChild.id)
+                          toggleUiMode(`edit-fridge-child-${fridgeChild.id}`)
+                        }}
+                        deletable={permittedActions.includes('DELETE')}
+                        onDelete={() => {
+                          setSelectedParentshipId(fridgeChild.id)
+                          toggleUiMode(`remove-fridge-child-${fridgeChild.id}`)
+                        }}
+                        conflict={fridgeChild.conflict}
+                        deletableFor={
```

Comment:
Is this still needed? What happens when both `deletable` and `deletableFor` are set?

Reply:
Removed this. Also fixed bug in `deletable` based on what's actually happening in `ParentshipController`.